src/mlpack/bindings/markdown/print_docs.cpp

```diff
@@ -256,7 +256,8 @@ void PrintDocs(const string& bindingName,
     }
 
     cout << "```" << BindingInfo::Language() << endl;
-    cout << example.substr(0, example.size() - 1) << endl; // do not want the last "\n".
+    cout << example.substr(0, example.size() - 1) << endl;
+    // do not want the last "\n".
```

Comment:
It is fine to have that, but I suppose the comment needs to be above as the linter moved it to the next line

Reply:
Good point, fixed in 4e14b782. :+1: